src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -197,6 +248,12 @@
       <OutputFolderForTestDependencies>$(BinDir)/TestDependencies</OutputFolderForTestDependencies>
     </PropertyGroup>
 
+    <!-- For UAP we need to copy the generated resources.pri containing the framework resources to the test folder -->
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true' AND '$(TestProjectNeedsModifiedPriFile)' != 'true'"
+          SourceFiles="$(CommonPriFile)"
```

Comment:
I can move that to that target and have the target executed every time and just condition it to either copy the common file or call the task to create a new one containing the tests resources.